docs/introduction.rst

```diff
@@ -17,9 +18,11 @@ such as addition and scalar multiplication.
 
 The following representations of intervals and pitches are implemented so far:
 
-* [spelled](types/spelled): based on Western notation
-* [enharmonic](types/enharmonic): based on the chromatic 12-tone system
-* [frequency](types/frequencies): based on the frequency of a sound wave
+* :doc:`spelled <types/spelled>`: based on Western notation
+* :doc:`spelled <types/spelled_array>`: based on Western notation
+* :doc:`enharmonic <types/enharmonic>`: based on the chromatic 12-tone system (WIP)
```

Comment:
I agree, it's good to have the disclaimer in and the more specific version now makes clear what to (not) expect. I have only removed the `WIP` as it suggests something being wrong or unreliable, which (from the Python library's point of view) is not the case. Of course, it would be nice to have full compatibility between the different versions of the library and we will hopefully achieve this in the future.